assets/src/edit-story/components/checklist/constants.js

```diff
@@ -80,6 +80,11 @@ export const ACCESSIBILITY_COPY = {
   },
   imagesMissingAltText: {
     title: __('Add assistive text to images', 'web-stories'),
+    // TODO: add `Learn more` link here to the footer?
```

Comment:
So actually, there's no learn more link here - that's in all of the designs in figma but there's actually no link we are keeping the content aligned with the spreadsheet not figma.

Reply:
Oh that's right! thanks for letting me know. I'll remove this